dvc/command/plot.py

```diff
@@ -0,0 +1,245 @@
+        "datafile",
+        nargs="?",
+        default=None,
+        help="Continuous metrics file to visualize.",
+    )
+    plot_show_parser.add_argument(
+        "-f", "--file", default=None, help="Name of the generated file."
+    )
+    plot_show_parser.add_argument(
+        "-s",
+        "--select",
+        default=None,
+        help="Choose which field(s) or JSONPath to include in the plot.",
+    )
+    plot_show_parser.add_argument(
+        "-x", default=None, help="Field name for x axis."
+    )
+    plot_show_parser.add_argument(
+        "-y", default=None, help="Field name for y axis."
+    )
+    plot_show_parser.add_argument(
+        "-o",
+        "--stdout",
+        action="store_true",
+        default=False,
+        help="Print plot specification to stdout.",
+    )
+    plot_show_parser.add_argument(
+        "--no-csv-header",
+        action="store_true",
+        default=False,
+        help="Required when CSV or TSV datafile does not have a header.",
+    )
+    plot_show_parser.add_argument(
+        "--no-html",
+        action="store_true",
+        default=False,
+        help="Do not wrap Vega plot JSON with HTML.",
+    )
+    plot_show_parser.add_argument("--title", default=None, help="Plot title.")
+    plot_show_parser.add_argument("--xlab", default=None, help="X axis title.")
+    plot_show_parser.add_argument("--ylab", default=None, help="Y axis title.")
+    plot_show_parser.set_defaults(func=CmdPlotShow)
+
+    PLOT_DIFF_HELP = (
+        "Plot continuous metrics differences between commits in the DVC "
+        "repository, or between the last commit and the workspace."
+    )
+    plot_diff_parser = plot_subparsers.add_parser(
+        "diff",
+        parents=[parent_parser],
+        description=append_doc_link(PLOT_DIFF_HELP, "plot/diff"),
+        help=PLOT_DIFF_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    plot_diff_parser.add_argument(
+        "-t",
+        "--template",
+        nargs="?",
+        default=None,
+        help="File to be injected with data.",
+    )
+    plot_diff_parser.add_argument(
+        "-d",
+        "--datafile",
+        nargs="?",
+        default=None,
+        help="Continuous metrics file to visualize.",
+    )
+    plot_diff_parser.add_argument(
+        "-f", "--file", default=None, help="Name of the generated file."
+    )
+    plot_diff_parser.add_argument(
+        "revisions",
+        nargs="*",
+        default=None,
+        help="Git revisions to plot from",
+    )
+    plot_diff_parser.add_argument(
+        "-s",
+        "--select",
+        default=None,
+        help="Choose which field(s) or JSONPath to include in the plot.",
+    )
+    plot_diff_parser.add_argument(
+        "-x", default=None, help="Field name for x axis."
+    )
+    plot_diff_parser.add_argument(
+        "-y", default=None, help="Field name for y axis."
+    )
+    plot_diff_parser.add_argument(
+        "-o",
```

Comment:
`-o` is usually associated with an `--output` that takes an argument. Feels weird to have it as `--stdout`